src/sage/modules/with_basis/indexed_element.pyx

```diff
@@ -27,6 +27,9 @@ from sage.misc.superseded import deprecation
 from sage.typeset.ascii_art import AsciiArt, empty_ascii_art, ascii_art
 from sage.typeset.unicode_art import UnicodeArt, empty_unicode_art, unicode_art
 from sage.data_structures.blas_dict cimport add, negate, scal, axpy
+from sage.categories.fields import Fields
+
+_Fields = Fields()
```

Comment:
I am not terribly fond of having another module-level memory-nail. I think it would be better to import this from a place like `rings/ring.pyx` or `categories/modules.py`.